manifests/init.pp

```diff
@@ -117,11 +117,6 @@
       'Container' => {
         'Image'   => $image,
         'Network' => 'host',
-        'Volume'  => [
-          "${log_dir}/uploads:/opt/app-root/src/uploads",
-          "${log_dir}/failed:/opt/app-root/src/failed",
-          "${log_dir}/logs:/opt/app-root/src/logs",
```

Comment:
Any reason why the volume mounts were removed?

Reply:
Essentially: permission problems (see https://github.com/theforeman/puppet-iop_advisor_engine/pull/10#discussion_r1926432120) and we seemed not to need it at that time.

If you need them back, we can work on it.

Reply:
Okay. The permission problem could be fixable...

I think it should be fine for now to have files exclusively inside a container. The deficiency of this is that the processed and failed archives would be lost with a container restarts.

cc @brantleyr

Reply:
Is there a process to clean up old data or will it grow unbounded?

Reply:
There is no process in iop-advisor-engine to clean up data. That should probably be an external job. I also don't think you'd want to blindly delete the failed archives and would probably want to analyze them before they are culled, so it may be desired to have additional metrics within the culling job.